src/app.js

```diff
@@ -17,6 +17,21 @@ const logger = require('../logs/logger');
 const app = express();
 require('express-async-errors');
 
+// Serve the Swagger JSON file
+// app.get('/swagger.json', (req, res) => {
+//   res.sendFile('swagger-output.json');
+// });
+
+// Serve Swagger UI static files
+app.use(
+  '/swagger-ui',
+  express.static(path.join(__dirname, 'node_modules/swagger-ui-dist'))
+);
+
+// Serve your generated Swagger specification
+app.get('/swagger', (req, res) => {
```

Comment:
maybe call this route `/api-docs.json` ?

Reply:
Renamed commit 262dab076f5099c97da0c7ee9c7969914bcdca62